.gitignore

```diff
@@ -2,6 +2,4 @@
 *.rej
 *.native
 _build
-setup.ml
-setup.data
-setup.log
+.merlin
```

Comment:
`jbuilder build @install` (makefile's `release` target) creates the `xen-api-sdk.install`, so yes, it seems I need it.